### ISSUE TITLE
Default plot method for osmdata class

### ISSUE THREAD

**Issue by Robinlovelace** [ropensci/osmdata#77]
Would be useful for users if this did not generate an error:

``` r
library(osmdata)
#> Data (c) OpenStreetMap contributors, ODbL 1.0. http://www.openstreetmap.org/copyright
q = add_feature(opq = opq("Ireland"), key = "left:nation", value = "Northern Ireland")
res = osmdata_sf(q = q)
class(res)
#> [1] "list"    "osmdata"
plot(res)
#> Error in xy.coords(x, y, xlabel, ylabel, log): 'x' is a list, but does not have components 'x' and 'y'
```

**Comment by mpadge** [comment]
Totally agree! Do you think it'd be worth adding `osmplotr` as a dependency and simply passing everything there? It'd make instant plots that would look quite nice.

**Comment by Robinlovelace** [comment]
Yes that sounds like a good solution.

**Comment by mpadge** [comment]
@Robinlovelace big things afoot in this dept! I'm working on a new package currently called [`svgplotr`](https://github.com/hypertidy/svgplotr). It includes a prototype for `svg` overlays directly on leaflet via [`d3`](https://github.com/d3/d3). I'll get a full-scale demo up hopefully tomorrow, but it scales and can do ... dunno, reckon at least 100,000's of lines. This will not only be the perfect default plot method for `osmdata`, but will be your ultimate solution for pct. Because they are vector objects, each line will also be able to be queried. It's gunna be big!

**Comment by mpadge** [comment]
Closed for the moment in favour of #155. A plot method might be implemented following the vignette, but first things first.
